app/build.gradle

```diff
@@ -64,21 +64,25 @@ android {
 }
 
 dependencies {
+
```

Comment:
can't comment on the right line but the following should be updated too:

id 'org.jetbrains.kotlin.plugin.serialization' version '1.8.10'